cobigen-templates/templates-devon4j/src/main/templates/crud_angular_client_app/templates/src/app/${variables.etoName#lower_case}/${variables.etoName#lower_case}-grid/${variables.etoName#lower_case}-grid.component.ts.ftl

```diff
@@ -83,20 +80,22 @@ export class ${variables.etoName?cap_first}GridComponent implements OnInit {
         (res: any) => {
           this.data = res.content;
           this.totalItems = res.totalElements;
-          this.dataTable.refresh();
         },
         (error: any) => {
           setTimeout(() => {
-            this._dialogService.openAlert({
-              message: error.message,
-              title: this.getTranslation('ERROR'),
-              closeButton: 'CLOSE',
+            this.dialog.open(${variables.etoName?cap_first}AlertComponent, {
+              width: '400px',
```

Comment:
Again here we should be careful with pixels.

Reply:
This one is applied only on larger devices. It gets a `max-width` style of `80vw` applied by Angular cdk layout so it looks fine on smaller devices as well.

Reply:
Okay, thanks for the explanation.